Marlin/Marlin_main.cpp

```diff
@@ -8281,29 +8271,33 @@ inline void gcode_M907() {
 
 #if HAS_CASE_LIGHT
 
-  uint8_t case_light_brightness = 255;
+  uint8_t case_light_brightness;
```

Comment:
If this were meant to determine the initialized value of `case_light_on` then it would be initialized here to something based on `CASE_LIGHT_DEFAULT_ON`. But, since `case_light_brightness` is being initialized in `setup()` and then `update_case_light` is being called, this is fine to leave as-is.